client/src/document/organisms/sidebar/index.tsx

```diff
@@ -16,11 +16,12 @@ export function RenderSideBar({ doc }) {
     if (doc.sidebarHTML) {
       return (
         <SidebarContainer>
+          <h4>Related Topics</h4>
           <span
```

Comment:
I don't know for sure but `<span>` is inline and, it might work, but it always feels a bit weird and wrong to use a inline element around a block element. That's what it was before.

Reply:
Yup, after thinking about it again, a `div` would make more sense. I was even considering changing it to a `nav` as that is essentially what the list is, but, then having a `nav` inside an `aside` seems wrong. What I am thinking, in the end, is that I am going to change the parent to a `nav` is that is more accurate of its contents, and change the `span` to a `div` as `span` was simply the wrong choice here.